From f7a97af23f85a5f49fbf887eefd397ec2d7b9d42 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:52 -0400
Subject: [redhat] kabi: Adding symbol clear_page_dirty_for_io (mm/page-writeback.c)

Message-id: <8bdb282179dccdde74986b6e962045481cebd3ca.1410510773.git.poros@redhat.com>
Patchwork-id: 95978
O-Subject: [PATCH RHEL7.1 052/233] kabi: Adding symbol clear_page_dirty_for_io (mm/page-writeback.c)
Bugzilla: 1071097
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Clear the dirty flag of page and update dirty memory accounting. Required to do write on a page.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/clear_page_dirty_for_io
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/clear_page_dirty_for_io
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/clear_page_dirty_for_io

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/clear_page_dirty_for_io b/redhat/kabi/kabi-module/kabi_ppc64/clear_page_dirty_for_io
new file mode 100644
index 0000000..45d0146
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/clear_page_dirty_for_io
@@ -0,0 +1,2 @@
+#1-
+0xb0aa81eb clear_page_dirty_for_io vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/clear_page_dirty_for_io b/redhat/kabi/kabi-module/kabi_s390x/clear_page_dirty_for_io
new file mode 100644
index 0000000..d7f3034
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/clear_page_dirty_for_io
@@ -0,0 +1,2 @@
+#1-
+0x602e32cc clear_page_dirty_for_io vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/clear_page_dirty_for_io b/redhat/kabi/kabi-module/kabi_x86_64/clear_page_dirty_for_io
new file mode 100644
index 0000000..e8e6a97
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/clear_page_dirty_for_io
@@ -0,0 +1,2 @@
+#1-
+0x2e1a09e1 clear_page_dirty_for_io vmlinux EXPORT_SYMBOL
-- 
1.7.1